server/server.js

```diff
@@ -1193,6 +1195,21 @@ exports.entryPage = "dashboard";
             console.log("Disabled Auth: auto login to admin");
             afterLogin(socket, await R.findOne("user"));
             socket.emit("autoLogin");
+        } else if (remoteUserHeader !== undefined) {
+            const remoteUser = socket.handshake.headers[remoteUserHeader.toLowerCase()];
+            if (remoteUser !== undefined) {
+                const user = await R.findOne("user", " username = ? AND active = 1 ", [
+                    remoteUser,
+                ]);
+                if (user) {
+                    afterLogin(socket, user);
+                    socket.emit("autoLogin");
+                } else {
+                    debug(`remote user ${remoteUser} doesnt exist`);
+                }
+            } else {
+                debug("remote user header set but not found in headers");
```

Comment:
```suggestion
                    debug(`Remote user ${remoteUser} doesn't exist`);
                }
            } else {
                debug("Remote user header set but not found in headers");
```

Reply:
Ah i though the debug logging was mostly lower level, but that seems to mostly be somewhat inconsistent.